pkgs/development/python-modules/pytorch-pfn-extras/default.nix

```diff
@@ -54,10 +52,24 @@ buildPythonPackage rec {
     # Requires CUDA access which is not possible in the nix environment.
     "-m 'not gpu and not mpi'"
     "-Wignore::DeprecationWarning"
+
+    # FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly...
+    "-Wignore::FutureWarning"
   ];
 
   pythonImportsCheck = [ "pytorch_pfn_extras" ];
 
+  disabledTests = [
+    # AssertionError: assert 4 == 0
+    # where 4 = <MagicMock id='140733587469184'>.call_count
+    "test_lr_scheduler_wait_for_first_optimizer_step"
+
+    # torch.distributed is not available on darwin
```

Comment:
Shouldn't this then be darwin specific?

Reply:
It should indeed. Fixed.

Reply:
The `test_lr_scheduler_wait_for_first_optimizer_step` should be skipped on all platforms though.